src/ostorlab/serve_app/types.py

```diff
@@ -120,6 +128,15 @@ def resolve_detail(
                 ],
             )
 
+    def resolve_risk_rating(
```

Comment:
Does this need unit test ?

Reply:
There is already one.. (independent from name of field type)